### PR TITLE
Update faillock related macros

### PR DESCRIPTION
**Pull request by Xeicker**
#### Description:
- Make faillock related macros manage the edge cases:
  - When the user had set options in pam files, but faillock.conf was available
  - When the option is not a number
  - Remove pam option when the option doesn't have a value associated

#### Rationale:

- As stated in `pam_faillock` man page, modifying pam_faillock options from pam files is still possible but not recommended.
- This prepares for the `dir` option which isn't a number
- Doing this it was found that when a pam option didn't have a value associated, the macro `bash_remove_pam_module_option_configuration` wasn't working

### CODE REVIEW

**Review by marcusburghardt — CHANGES_REQUESTED** [review]
Thanks @Xeicker . I just included some small comments.

File: shared/macros/10-bash.jinja [hunk 1]
```diff
@@ -1037,11 +1038,14 @@ if [ -f $FAILLOCK_CONF ]; then
     fi
     {{%- else %}}
     else
-        sed -i --follow-symlinks 's/^\s*\({{{ option }}}\s*=\s*\)\([0-9]\+\)/\1'"{{{ value }}}"'/g' $FAILLOCK_CONF
+        sed -i --follow-symlinks 's|^\s*\({{{ option }}}\s*=\s*\)\(\S\+\)|\1'"{{{ value }}}"'|g' $FAILLOCK_CONF
```
[the remark this answers is not in the document]
Comment (Xeicker) [reply shared/macros/10-bash.jinja:1041]: this was thinking in the `dir` option, which could use non-alpha-numeric characters.

[the remark this answers is not in the document]
Comment (marcusburghardt) [reply shared/macros/10-bash.jinja:1043]: Never mind. I forgot this case. : )

**Review by marcusburghardt — APPROVED** [review]
Thanks @Xeicker . I will just wait the tests to finish.

**Comment by marcusburghardt** [comment]
Besides the CI tests I also tested critical PAM rules in local VMs and everything worked fine. Thanks @Xeicker

**Pull request merged**